### ISSUE TITLE
LaTeX documentation build failure.

### ISSUE THREAD

**Issue by cgranade** [QInfer/python-qinfer#94]
When using Sphinx to build the PDF User's Guide, the compilation fails due to the API documentation for the new ``in_credible_region``:

```
Underfull \hbox (badness 10000) in paragraph at lines 6178--6180
[]\T1/pcr/b/n/10 level \T1/ptm/m/n/10 ([][]\T1/pcr/m/sl/10 float[][]\T1/ptm/m/n
/10 ) -- The de-sired cred-i-b-lity level (see

! LaTeX Error: Too deeply nested.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...

l.6186 \item {} \begin{description}
```

**Comment by ihincks** [comment]
My bad, the sphinx string for ``region_est_ellipsoid`` should be ``r""""""`` not ``""""""``. I'm not sure if there are more problems or not.

**Comment by cgranade** [comment]
Turns out to be a couple unrelated issues; should be fixed and regression tested in #95 soon.
